lib/js/components/Layout/Layout.consts.ts

```diff
@@ -0,0 +1,22 @@
+export const LAYOUT_CONTENT_COLUMN_SIZE = {
+	FULL_WIDTH: 'fullWidth',
+	MAX_900: 'max900',
```

Comment:
using pixel value in const name requires migration of all component usages when we decide to change size. IMO, we should avoid that